packages/devtools_shared/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Package of shared structures between devtools_app and devtools_serv
 
 # Note: this version should only be updated by running tools/update_version.sh
 # that updates all versions of packages from packages/devtools.
-version: 0.2.3
+version: 0.2.4
```

Comment:
don't update this version. we'll update all the versions in lock step when we publish.

Reply:
Yeah, I wasn't sure if I needed to or not. Reverted here and elsewhere.